src/reactionHandlers/updateReactboard.ts

```diff
@@ -0,0 +1,188 @@
+import {
+	Attachment,
+	ChannelType,
+	DMChannel,
+	EmbedBuilder,
+	Message,
+	MessageReaction,
+	NewsChannel,
+	PartialDMChannel,
+	PrivateThreadChannel,
+	PublicThreadChannel,
+	TextChannel,
+	VoiceChannel,
+} from 'discord.js';
+import { db } from '../database';
+import { appVersion } from '../constants/meta';
+
+export const updateReactboard: ReactionHandler = {
+	async execute({ reaction, user }) {
+		const fullReaction = await reaction.fetch();
+		const fullMessage = await reaction.message.fetch();
+		const fullUser = await user.fetch();
```

Comment:
There's a good chance that the reaction and user data aren't partial, so doing a full `fetch()` on these would be needless overhead.

```suggestion
		const fullReaction = reaction.partial ? await reaction.fetch() : reaction;
		const fullMessage = reaction.message.partial
			? await reaction.message.fetch()
			: reaction.message;
		const fullUser = user.partial ? await user.fetch() : user;
```